grails-app/views/purchaseOrder/_showOrderItems.gsp

```diff
@@ -394,6 +392,17 @@
           });
         }
 
+        function getTotalAdjustments() {
```

Comment:
This doesn't feel very DRY :) 

We could probably add a utility library for methods like this. I have the same or very similar code in a few places in _productDetails.gsp. Perhaps we could make one single function in a utils.js library that does the following

```
function refreshDataPoint(url, data, target, digits) { 
    $.ajax({
        url:url,
        data: data,
        success: function(data, textStatus){
              target.html(parseFloat(data).toFixed(digits));
        }
    });
}
```

And then on pages like this we'd just use the library method.
```
function getTotalAdjustments() { 
   var target = $("#totalAdjustments");
   var data = { orderId: $("#orderId").val() }
   var url = "${g.createLink( controller:'order', action:'getTotalAdjustments')}"
   refreshDataPoint(url, data, target, 2)
}
```

In this case, I understand this doesn't really save us all that much, but it'll be much easier to troubleshoot and extend if it's all in one place.

Reply:
@jmiranda I think it's a good idea. I'll create a new ticket for that. Does that sounds good?